targets/XPlatCppSdk/source/code/source/playfab/PlayFabWinHttpPlugin.cpp

```diff
@@ -118,177 +118,183 @@ namespace PlayFab
-                                        if (!WinHttpQueryDataAvailable(hRequest, &dwSize))
-                                        {
-                                            SetErrorInfo(reqContainer, "Error in WinHttpQueryDataAvailable, failed to check for an available data block in HTTP response");
-                                            bResults = FALSE;
-                                            break;
-                                        }
-
-                                        // Allocate space for the buffer
-                                        auto outBuffer = std::unique_ptr<char>(new char[dwSize + 1]);
-                                        if (!outBuffer)
-                                        {
-                                            SetErrorInfo(reqContainer, "Out of memory, failed to allocate a buffer to read a data block in HTTP response");
-                                            bResults = FALSE;
-                                            break;
-                                        }
-                                        else
-                                        {
-                                            // Read the data block
-                                            ZeroMemory(outBuffer.get(), dwSize + 1);
-                                            if (!WinHttpReadData(hRequest, (LPVOID)outBuffer.get(), dwSize, &dwDownloaded))
-                                            {
-                                                SetErrorInfo(reqContainer, "Error in WinHttpReadData, failed to read a data block in HTTP response");
-                                                bResults = FALSE;
-                                                break;
-                                            }
-                                            else
-                                            {
-                                                // successfully received a block of data
-                                                reqContainer.responseString.append(outBuffer.get());
-                                            }
-
-                                            // Free the memory allocated to the buffer
-                                            outBuffer = nullptr;
-                                        }
-
-                                    } while (dwSize > 0);
-
-                                    if (bResults)
-                                    {
-                                        ProcessResponse(reqContainer, dwStatusCode);
-                                    }
-                                } // WinHttpQueryHeaders
-                            } // WinHttpReceiveResponse
-                        } // WinHttpSendRequest
-                    } // WinHttpOpenRequest
-                } // WinHttpConnect
-            } // WinHttpOpen
-        } // WinHttpCrackUrl
-
+		if (PlayFabSettings::verticalName.empty() && PlayFabSettings::titleId.empty())
```

Comment:
Let's remove this condition:
PlayFabSettings::verticalName.empty() && 
The verticalName is always going to be empty right now (that feature isn't finished).